.github/workflows/ci.yml

```diff
@@ -28,37 +28,37 @@ jobs:
 
     strategy:
       matrix:
-        eslint: [6.x]
-        node: [14.x]
+        eslint: [7]
+        node: [14]
```

Comment:
I think this change (deleting `.x`) is unrelated to confirming ESLint 7 support. It's better if each PR contains only related changes.

Reply:
Will do that in a separate PR then